actionpack/test/dispatch/live_response_test.rb

```diff
@@ -65,7 +65,7 @@ def test_headers_cannot_be_written_after_webserver_reads
         latch = Concurrent::CountDownLatch.new
 
         t = Thread.new {
-          @response.stream.each do
```

Comment:
I think this behavior needs to remain unchanged. In a streaming response, we should only be able to call each on the stream once.